### PR TITLE
Fix rtd

### PR DESCRIPTION
**Pull request by bardkw**
Fix documentation for  SMQTK-Image-IO

### CODE REVIEW

**Comment by Purg** [comment]
@bardkw With the merger of the restructure branch, please rebase this.

File: docs/sphinx_server.py [hunk 1]
```diff
@@ -18,25 +18,25 @@
...
 for d in watch_dirs:
     for g in watch_globs:
         server.watch(os.path.join(d, g), rebuild_cmd)
 # Watch source python files.
-for dirpath, dirnames, filenames in os.walk('../python/smqtk'):
+for dirpath, dirnames, filenames in os.walk(watch_source_dir):
     server.watch(os.path.join(dirpath, '*.py'), rebuild_cmd)
 # Optionally change to host="0.0.0.0" to make available outside localhost.
-server.serve(root='_build/html')
+server.serve(root=rebuild_root)
```
Comment (Purg) [inline docs/sphinx_server.py:42]: This variable needs to be defined above.

File: docs/release_notes/pending_release.rst [hunk 2]
```diff
@@ -5,10 +5,17 @@ SMQTK Pending Release Notes
 Updates / New Features
 ----------------------
 
-CI
```
Comment (Purg) [inline docs/release_notes/pending_release.rst:8]: Why remove the CI note?

**Review by Purg — CHANGES_REQUESTED** [review]
Looks mostly good. Some relatively minor text cleanup and working things. It seems like the pending change note added clobbers the existing one about adding the github action workflow?

**Comment by Purg** [comment]
This PR should also be renamed to remove the "WIP" prefix.

**Comment by Purg** [comment]
Checking the RTD preview build shows that the PIL/GDAL impl ref page is not rendering correctly: https://smqtk-image-io--3.org.readthedocs.build/en/3/image_io_implementations.html

File: docs/image_io_implementations.rst [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Implementations
+---------------
+
+Image Reader implementations that use PIL and GDAL.
+
+PilImageReader
+++++++++++++++
+
+.. autoclass:: smqtk_image_io.impls.image_reader.pil_io.PilImageReader
+
```
Comment (Purg) [inline docs/image_io_implementations.rst:10]: This extra new-line is causing issues sphinx parsing for rendering.

Comment (bardkw) [reply docs/image_io_implementations.rst:10]: really? without it I'm getting this error `SMQTK-Image-IO/smqtk_image_io/impls/image_reader/pil_io.py:docstring of smqtk_image_io.impls.image_reader.pil_io.PilImageReader.valid_content_types:2: WARNING: Field list ends without a blank line; unexpected unindent.`

Comment (Purg) [reply docs/image_io_implementations.rst:10]: I see that too, but this additional new-line is also an issue. It doesn't show a warning, it just gets parsed in a way that we don't want it to be.

Comment (Purg) [reply docs/image_io_implementations.rst:10]: I think that warning may stem from the parent class doc-string, so we can ignore that here I think.

**Comment by Purg** [comment]
A thing I just noticed is that the `_`-prefixed abstract method is not being shown in the interface/impl docs. I think that is because the `:members:` directive does not include "private" members. There should be an option for the `.. code-block::` thing that allows the exposure of those. Basically we want that abstract method to show up in the reference.

**Review by Purg — APPROVED** [review]
It's weird that the RTD preview doesn't seem to render anything for the impls page but a local build does just fine. I guess we'll see what RTD does in an "official" build, I guess.

**Pull request merged**